### DISCUSSION TITLE
分秒帧的cookies在7天后会自动更新吗？

### DISCUSSION THREAD

**Discussion by fzlychee2019** [AlistGo/alist#1057] [General]
分秒帧的cookies有效期只有7天，那系统能在7天后自动更新吗？看介绍上说分秒帧的分享有一天、七天、永久三种，不能用上永久的分享功能吗？如果每7天得重新修改一次cookies，那就太麻烦了 ，也就失去使用价值了。期待官方出解决方法。

**Comment by wuhenge** [comment]
账户会被封禁的，我用了几天账户就被封禁了

**Reply by fzlychee2019** [reply]
这么严重？不会吧？

**Reply by wuhenge** [reply]
这还能有假，https://pic.rmb.bdstatic.com/bjh/02010b3f803644de4a03f5a4e568bc7c.jpeg
